eslint.config.js

```diff
@@ -9,26 +9,18 @@ import tseslint from 'typescript-eslint';
 const todoTreeKeywordsWarning = ['TODO', 'TODOS', 'TODO WIP', 'FIXME', 'WIP'];
 const todoTreeKeywordsAll = [...todoTreeKeywordsWarning, 'NOTE', 'NOTES', 'LIST'];
 
-const OFF = 0;
```

Comment:
This change appears to go in the opposite direction to what I would expect: You are replacing nicely factored out constants (numbers) by "magic strings": 'warn', 'off', 'error'. Why can't these remain constants?

Reply:
I understand that if we use 0,1, or 2 they'd be magic constants, but ESLint lets directly use `off`, `warn` and `error`, which are self-explanatory. I don't see the need for this extra indirection.

Reply:
Yes, I agree. Not much difference between having strings and identifiers that refer to strings.